### PR TITLE
Add implementation to get short id

### LINKED ISSUE #26: Ability to get short id from store.getId() using a flag
We can introduce a flag to make it easy to get short id from `store.getId()`. Something like this:
```ts
function getId(short: boolean = false) {
  // By default it returns the regular result. But if `short = true` is passed it returns the initial 8 characters of the id. 
  ...
}
```
**Usage:**
Example: https://github.com/leapfrogtechnology/async-store/blob/e899695eedf9aa6aa6cbb256f0618591396c08da/examples/express-ts/src/logger.ts#L9

```diff
- return (store.getId() || '').substring(0, 8); 
+ return store.getId(true);
```

### PR DESCRIPTION
**Pull request by silwalanish**
Resolves #26 

- Add `getShortId` method to get short unique domain id.

### CODE REVIEW

**Comment by silwalanish** [comment]
@kabirbaidhya can you review this?

File: src/impl/domain.ts [hunk 1]
```diff
@@ -234,6 +234,18 @@ export function getId(): string | undefined {
...
 }
 
+/**
+ * Gets the short unique domain id created for the current context / scope.
+ *
+ * @returns {(string | undefined)}
+ */
+export function getShortId(): string | undefined {
+  const activeDomain = getActiveDomain();
+  const id = activeDomain && activeDomain[ID_KEY];
```
Comment (kabirbaidhya) [inline src/impl/domain.ts]: No need to duplicate the code when you could do:
```suggestion
  const id = getId();
```

File: src/impl/domain.ts [hunk 2]
```diff
@@ -234,6 +234,18 @@ export function getId(): string | undefined {
...
+
```
Comment (kabirbaidhya) [inline src/impl/domain.ts]: No need to duplicate the code when you could do:
```suggestion
  const id = getId();
```

File: README.md [hunk 3]
```diff
@@ -227,6 +227,16 @@ Example: If used in express, it returns unique store id per request.
 const requestIdentifier = store.getId();
 ```
 
+### getShortId()
+
+Gets the short unique store id created for the current context/scope.
+
```
Comment (kabirbaidhya) [inline README.md]: Also add a line saying - This is same as `getId()`; the difference being just that it returns the id of shorter length (8 characters).

**Pull request merged**